Tests/FluentAssertions.Specs/AssertionOptionsSpecs.cs

```diff
@@ -10,395 +10,391 @@
 using Xunit;
 using Xunit.Sdk;
 
-namespace FluentAssertions.Specs
+namespace FluentAssertions.Specs;
+
+public class AssertionOptionsSpecs
 {
-    namespace AssertionOptionsSpecs
```

Comment:
Changed namespace from `FluentAssertions.Specs.AssertionOptionsSpecs` to `FluentAssertions.Specs`